### PR TITLE
changes from our discussion in the ios guild

### PR DESCRIPTION
**Pull request by bensNL**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,12 +122,11 @@ if n.isNumber {
...
 
 1. Extensions should be named for the sort of functionality they provide. Don't create umbrella extensions, eg. "Helper".
-2. Accessibility should be defined on the extension and not on it's methods.
 
 ## **K)** Naming ##
 
 1. Always use descriptive names instead of shortcuts (readability over speed).
 2. A method declaration should read like a sentence and be absolutely clear about what it does, what it needs and what it returns.
-3. `IBOutlets` should end with their element class name and be private. If you need to access the outlet in an extension use fileprivate instead.
+3. `IBOutlets` should end with their element class name and be private. If you need to access the outlet in an extension use fileprivate instead. Do not use `weak` for `IBOutlets` they should be strong in Swift, which is the default.
```
Comment (FranzBusch) [inline README.md]: I would prefer to rephrase it to they should be strong. Because 1. it is not Swift dependent and 2. the default when you install Xcode is still weak (unless I am mistaken)

Comment (FranzBusch) [reply README.md]: https://stackoverflow.com/a/31395938/3960382

Comment (JesusAdolfo) [reply README.md]: You're right, for some reason weak is the default if you create the outlet with IB

Comment (FranzBusch) [reply README.md]: The reason is that at the beginning of iOS, there was viewDidLoad and viewDidUnload and if your outlets were strong you had to deference them there otherwise there would be a retain cycle. Since then they removed this cycle and there is no reason to have them weak anymore

File: README.md [hunk 2]
```diff
@@ -118,12 +122,11 @@ if n.isNumber {
 ## **J)** Extensions ##
 
 1. Extensions should be named for the sort of functionality they provide. Don't create umbrella extensions, eg. "Helper".
-2. Accessibility should be defined on the extension and not on it's methods.
```
Comment (kimlanbui) [inline README.md]: Accessibility should be defined on the method itself and not on the extension.

Comment (bensNL) [reply README.md]: but do we really need that here?

**Comment by bensNL** [comment]
updated the PR according to the comments. please check again :)

File: README.md (hunk 2 above)
Comment (bensNL) [reply README.md]: I took this change out of the PR since there are different opinions about it and I do not want to block the rest of the PR with it.

**Comment by bensNL** [comment]
Please review the PR again. Accessibility for extensions will be discussed in another PR, I got that wrong in the first place, sorry.
And just as a reminder: the changes in this PR have already been discussed in a guild and were the agreed outcome ;)
